### PR TITLE
feat: Add update icon

### PR DESCRIPTION
**Pull request by ashik-v**
Add an update icon.

### CODE REVIEW

File: assets/.DS_Store
Comment (haideralsh) [inline assets/.DS_Store:1]: Is this added on purpose?

File: assets/icons.json [hunk 1]
```diff
@@ -312,6 +312,12 @@
     ],
     "viewBox": "0 0 24 24"
   },
+  "update": {
+    "path": [
+      "M483 936q-75 0-141-28.5T226.5 830q-49.5-49-78-115T120 574q0-75 28.5-140t78-113.5Q276 272 342 244t141-28q80 0 151.5 35T758 347V241h60v208H609v-60h105q-44-51-103.5-82T483 276q-125 0-214 85.5T180 571q0 127 88 216t215 89q125 0 211-88t86-213h60q0 150-104 255.5T483 936Zm122-197L451 587V373h60v189l137 134-43 43Z"
+    ],
+    "viewBox": "0 96 960 960"
```
Comment (haideralsh) [inline assets/icons.json:319]: Is there a reason the viewBox of the svg looks different that the other ones in the file `0 0 24 24`?

File: assets/update.svg
Comment (haideralsh) [inline assets/update.svg:1]: The icon here seems to be `48px by 48px` I think we need to resize it to be like the rest of the icons `24px by 24px`.

**Review by haideralsh — APPROVED** [review]

File: assets/.DS_Store
Comment (ashik-v) [reply assets/.DS_Store:1]: It was there after we ran the build command; we can remove it and git ignore it if you think it's not required.

File: assets/icons.json (hunk 1 above)
Comment (ashik-v) [reply assets/icons.json:319]: No reason - we got the file and it was like this. We can reduce it

File: assets/update.svg
Comment (ashik-v) [reply assets/update.svg:1]: We can reduce the size

**Review by haideralsh — APPROVED** [review]

**Pull request merged**